onnxruntime/test/python/transformers/test_flash_attn_cuda.py

```diff
@@ -1343,59 +1325,14 @@ def parity_check_gqa_prompt(
     out = out.detach().cpu().numpy()
 
     # Make sure past-present buffer updating correctly
-    assert numpy.allclose(present_k, k_cache_ref.detach().cpu().numpy(), rtol=rtol, atol=atol, equal_nan=True)
-    assert numpy.allclose(present_v, v_cache_ref.detach().cpu().numpy(), rtol=rtol, atol=atol, equal_nan=True)
-
-    # Compare results
-    all_close = numpy.allclose(out, out_ref, rtol=rtol, atol=atol, equal_nan=True)
-    correct = GREEN + "True" + RESET if all_close else RED + "False" + RESET
-    print(
-        "KV-buffer",
-        " packed:",
-        packed,
-        " causal:",
-        causal,
-        " local:",
-        local,
-        " rotary:",
-        rotary,
-        " rotary_interleaved:",
-        rotary_interleaved,
```

Comment:
Could you move these parameters into either Config or add them to error message? Otherwise, they will not show up in error message

Reply:
Done